_sass/minimal-mistakes/_variables.scss

```diff
@@ -61,6 +61,7 @@ $muted-text-color: mix(#fff, $text-color, 35%) !default;
 $border-color: $lighter-gray !default;
 $form-background-color: $lighter-gray !default;
 $footer-background-color: $lighter-gray !default;
+$active-color: mix($light-gray, $lighter-gray, 50%) !default;
```

Comment:
Switch to the following and move after `$primary-color` line to avoid Sass error.

```scss
$active-color: mix(#fff, $primary-color, 80%) !default;
```